src/qt/rpcconsole.cpp

```diff
@@ -471,14 +471,28 @@ RPCConsole::RPCConsole(interfaces::Node& node, const PlatformStyle *_platformSty
 
     constexpr QChar nonbreaking_hyphen(8209);
     const std::vector<QString> CONNECTION_TYPE_DOC{
+        //: Explanatory text for an Inbound peer connection.
         tr("Inbound: initiated by peer"),
+        /*: Explanatory text for an Outbound peer connection which
+            relays all network information. This is the default behavior for
+            Outbound connections. */
         tr("Outbound Full Relay: default"),
+        /*: Explanatory text for an Outbound peer connection which
+            only relays network information about Blocks. */
```

Comment:
For translators it is not obvious the information relayed through the network is related to blocks, transactions, addresses, and service needs (handshaking, ping etc). _"only relays network information about Blocks"_ is not the same as _"does not relay transactions or addresses"_ to them. Btw, the latter seems more correct as `BLOCK_RELAY` peers handle service network commands well.

Reply:
Yes. 

(It's not clear to me what value these descriptions add, as they are next to descriptions that already exist.)

Reply:
> It's not clear to me what value these descriptions add, as they are next to descriptions that already exist.

If understand your concern correctly, the point is that in Transifex.com on-line editor a translator can see only a translatable string and an attached comment, without surrounding code.

Reply:
Thanks for the explanation. That makes sense.